composer.json

```diff
@@ -24,11 +24,12 @@
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
-        "doctrine/phpcr-odm": "^1.4 || ^2.0",
+        "doctrine/phpcr-odm": "^1.4",
```

Comment:
is this an expected change?

Reply:
it is the same issue you observed: phpcr-bundle 2 and phpcr-odm 2 are not compatible.

but let me try if phpcr-bundle 3 would be difficult to support in a minor version (i think it should be doable)